source/java/spring-boot-web/tutorial.html

```diff
@@ -0,0 +1,845 @@
+<li><code class="docutils literal"><span class="pre">C:\Users\YOUR_USERNAME\.stormpath\apiKey.properties</span></code> on Windows</li>
+</ul>
+</li>
+<li><p class="first">Change the file permissions to ensure only you can read this file and not accidentally write or modify it. For example:</p>
+<blockquote>
+<div><div class="highlight-bash"><div class="highlight"><pre><span class="nv">$ </span>chmod go-rwx ~/.stormpath/apiKey.properties
+<span class="nv">$ </span>chmod u-w ~/.stormpath/apiKey.properties
+</pre></div>
+</div>
+</div></blockquote>
+</li>
+</ol>
+<p>On Windows, you can <a class="reference external" href="http://msdn.microsoft.com/en-us/library/bb727008.aspx">set file permissions similarly</a>.</p>
+</div>
+<div class="section" id="add-the-stormpath-default-spring-boot-starter">
+<span id="dependency-jar"></span><h2><a class="toc-backref" href="#id17">Add the Stormpath Default Spring Boot Starter</a><a class="headerlink" href="#add-the-stormpath-default-spring-boot-starter" title="Permalink to this headline">¶</a></h2>
+<p>This step allows you to enable Stormpath in a Spring Boot web app <em>without a single line of code or configuration</em>.
+It includes Stormpath Spring Security, Stormpath Spring WebMVC and Stormpath Thymeleaf templates. How amazing is that? Here&#8217;s how.</p>
+<p>Using your favorite dependency resolution build tool like Maven or Gradle, add the stormpath-default-spring-boot-starter-1.0.RC5-SNAPSHOT.jar to your project dependencies. For example:</p>
+<p><strong>Maven</strong>:</p>
+<pre class="literal-block">
+&lt;dependency&gt;
+    &lt;groupId&gt;com.stormpath.spring&lt;/groupId&gt;
+    &lt;artifactId&gt;stormpath-default-spring-boot-starter&lt;/artifactId&gt;
+    &lt;version&gt;1.0.RC5-SNAPSHOT&lt;/version&gt;
+&lt;/dependency&gt;
+</pre>
+<p><strong>Gradle</strong>:</p>
+<pre class="literal-block">
+dependencies {
+    compile 'com.stormpath.spring:stormpath-default-spring-boot-starter:1.0.RC5-SNAPSHOT'
+}
+</pre>
+<p>That&#8217;s it!  You&#8217;re ready to start using Stormpath in your Spring Boot web application!  Can you believe how easy that was?</p>
+</div>
+<div class="section" id="spring-boot-meet-stormpath">
+<span id="id2"></span><h2><a class="toc-backref" href="#id18">Spring Boot: Meet Stormpath</a><a class="headerlink" href="#spring-boot-meet-stormpath" title="Permalink to this headline">¶</a></h2>
+<p>Let&#8217;s fire up a basic Spring Boot Web application. The code for this section can be found <a class="reference external" href="https://github.com/stormpath/stormpath-sdk-java/tree/master/tutorials/spring-boot-default/00-the-basics">here</a>.</p>
+<p>For a maven build and run, do this:</p>
+<div class="highlight-bash"><div class="highlight"><pre>mvn clean package
+
+<span class="nv">STORMPATH_API_KEY_FILE</span><span class="o">=</span>&lt;path to your key file&gt; <span class="se">\</span>
```

Comment:
You can also run it this way: `mvn spring-boot:run`. I think I like it better since it is shorter and they are already using maven in the previous line.